gammapy/irf/effective_area_table.py

```diff
@@ -360,10 +370,27 @@ class EffectiveAreaTable2D(object):
     >>> from gammapy.datasets import load_aeff2D_fits_table
     >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
     >>> offset = Angle(0.6, 'degree')
-    >>> energies = Quantity(np.logspace(0, 1, 60), 'TeV')
-    >>> eff_area = aeff2D.evaluate(offset, energies)
+    >>> energy = Quantity(np.logspace(0, 1, 60), 'TeV')
+    >>> eff_area = aeff2D.evaluate(offset, energy)
 
-    Plot energy dependence 
+    Create ARF fits file for a given offest and energy binning:
+
+    >>> import numpy as np
+    >>> from astropy.coordinates import Angle
+    >>> from astropy.units import Quantity
+    >>> from gammapy.irf import EffectiveAreaTable2D
+    >>> from gammapy.spectrum import energy_bounds_equal_log_spacing
+    >>> from gammapy.datasets import load_aeff2D_fits_table
+    >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
+    >>> offset = Angle(0.43, 'degree')
+    >>> nbins = 50
+    >>> energy = energy_bounds_equal_log_spacing(Quantity((1,10), 'TeV', nbins)
```

Comment:
I just tried this example locally.
There's a missing closing parens on this line.

If I add it, I get this error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-55b7c69c7675> in <module>()
      8 offset = Angle(0.43, 'degree')
      9 nbins = 50
---> 10 energy = energy_bounds_equal_log_spacing(Quantity((1,10), 'TeV', nbins))
     11 energ_lo = energy[:-1]
     12 energ_hi = energy[1:]

/Users/deil/Library/Python/3.4/lib/python/site-packages/astropy-1.1.dev12994-py3.4-macosx-10.10-x86_64.egg/astropy/units/quantity.py in __new__(cls, value, unit, dtype, copy, order, subok, ndmin)
    247 
    248         value = np.array(value, dtype=dtype, copy=copy, order=order,
--> 249                          subok=False, ndmin=ndmin)
    250 
    251         # check that array contains numbers or long int objects

TypeError: data type not understood
```

@kingj90 – Can you fix this example as a separate commit in your next PR?

Also: code examples that don't show output don't have to have the `>>>` on every line, you can remove that.